### PR TITLE
Update Dutch translation (preview)

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
- Stubs PR: https://github.com/microbit-foundation/micropython-microbit-stubs/pull/86
- Pyright PR: https://github.com/microbit-foundation/pyright/pull/28

### CODE REVIEW

File: lang/ui.nl.json [hunk 1]
```diff
@@ -611,7 +611,7 @@
     "description": "Aria label for the redo button"
   },
   "reference-tab": {
-    "defaultMessage": "Referentie",
+    "defaultMessage": "Sommige van je wijzigingen zijn niet opgeslagen. Toch afsluiten?",
```
Comment (microbit-matt-hillsdon) [inline lang/ui.nl.json:614]: We can't merge this change!

From

image

To

image

Comment (microbit-matt-hillsdon) [reply lang/ui.nl.json:614]: Commented in Crowdin here: https://crowdin.com/translate/microbitorg/6426/en-nl?filter=basic&value=0#q=reference

The change was very recent.

**Comment by microbit-matt-hillsdon** [comment]
Moved back to a draft until I resync after a Crowdin fix.

File: lang/ui.nl.json (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply lang/ui.nl.json]: db5fe1f821e0ba16b8ae63ae002948f893dab130

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**